cmd/minishift/cmd/start.go

```diff
@@ -779,3 +784,17 @@ func cacheMinishiftISO(config *cluster.MachineConfig) {
 func shouldPreflightChecksBeSkipped() bool {
 	return viper.GetBool(configCmd.SkipPreflightChecks.Name)
 }
+
+func getOpenShiftVersion() (string, error) {
+	tag := viper.GetString(configCmd.OpenshiftVersion.Name)
+	if tag == "vlatest" {
+		tags, err := openshiftVersion.GetSortedOpenShiftTags(constants.MinimumSupportedOpenShiftVersion, version.GetOpenShiftVersion())
```

Comment:
I always dislike names that start with `Sorted` as it means nothing without more context. Which way is it sorted, ascending or descending? if an option would have been added, this is clear... or rename the function to include the actual sort order. `GetOpenShiftVersionByTagsAscending` or similar.